apps/docs/pages/guides/auth/passwordless-login/auth-email-otp.mdx

```diff
@@ -0,0 +1,121 @@
+- Navigate to the [email template settings](/dashboard/project/_/auth/templates) and modify the template to include the `{{ .Token }}` variable, for example:
+
+```html
+<h2>One time login code</h2>
+
+<p>Please enter this code:{{ .Token }}</p>
+```
+
+## Sign in with email OTP
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="js"
+  queryGroup="language"
+>
+<TabPanel id="js" label="JavaScript">
+When your user signs in, call [signInWithOtp()](/docs/reference/javascript/auth-signinwithotp) with their email address:
+
+```js
+const { data, error } = await supabase.auth.signInWithOtp({
+  email: 'example@email.com',
+  options: {
+    // set this to false if you do not want the user to be automatically signed up
+    shouldCreateUser: false,
+  },
+})
+```
+
+</TabPanel>
+<TabPanel id="dart" label="Dart">
+
+When your user signs in, call [signIn()](/docs/reference/dart/auth-signinwithotp) with their email address:
+
+```dart
+Future<void> signInWithEmailOtp() async {
+  final AuthResponse res = await supabase.auth.signinwithotp(email: 'example@email.com');
+}
+```
+
+</TabPanel>
+<TabPanel id="kotlin" label="Kotlin">
+
+When your user signs in, call [sendOtpTo()](/docs/reference/kotlin/auth-signinwithotp) with their email address:
+
+```kotlin
+suspend fun signInWithEmailOtp() {
+	supabase.gotrue.sendOtpTo(Email) {
+		email = "example@email.com"
+	}
+}
+```
+
+</TabPanel>
+</Tabs>
+
+If the request was successful, you receive a response with `error: null` and a `data` object where both `user` and `session` are null. In this case you can let the user know to check their email inbox.
+
+```json
+{
+  "data": {
+    "user": null,
+    "session": null
+  },
+  "error": null
+}
+```
+
+## Verify OTP to create session
+
+Provide an input field for the user to key in the one-time code. To verify the code and complete the user's sign in, call [verifyOtp()](/docs/reference/javascript/auth-verifyotp) with their email address, the code, and `type: "email"`:
+
+```js
+const {
+  data: { session },
+  error,
+} = await supabase.auth.verifyOtp({
+  email,
+  token: '123456',
+  type: 'email',
+})
+```
+
+If successful the user will now be logged in and you should receive a valid session like:
+
+```json
+{
+  "access_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNjI3MjkxNTc3LCJzdWIiOiJmYTA2NTQ1Zi1kYmI1LTQxY2EtYjk1NC1kOGUyOTg4YzcxOTEiLCJlbWFpbCI6IiIsInBob25lIjoiNjU4NzUyMjAyOSIsImFwcF9tZXRhZGF0YSI6eyJwcm92aWRlciI6InBob25lIn0sInVzZXJfbWV0YWRhdGEiOnt9LCJyb2xlIjoiYXV0aGVudGljYXRlZCJ9.1BqRi0NbS_yr1f6hnr4q3s1ylMR3c1vkiJ4e_N55dhM",
+  "token_type": "bearer",
+  "expires_in": 3600,
+  "refresh_token": "LSp8LglPPvf0DxGMSj-vaQ",
+  "user": {...}
+}
+```
+
+## Resources
+
+- [Supabase Account - Free Plan OK](https://supabase.com)
```

Comment:
Is this the intended link?

Reply:
I actually don't know where these are coming from. I *think* their purpose is to show that you can use this feature with a free Supabase account?

Reply:
I think if we want to say that we should just add a sentence somewhere to say it 😄 and take out this link as it's confusing